src/project-css/app/Jobs/UpdateSearchIndex.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Jobs;
+
+use App\Adapters\UpdateSearchAdapter;
+use Illuminate\Bus\Queueable;
+use Illuminate\Http\Request;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Support\Facades\DB;
+
+use Log;
+
+class UpdateSearchIndex implements ShouldQueue
+{
+    use InteractsWithQueue, Queueable, SerializesModels;
```

Comment:
How are these used?

Reply:
These are generally required traits used in jobs. I always include them (fails without them I tested it once). It is explained in the site below if you want to read about it.

https://blog.tiger-workshop.com/laravel-queue-job-class-demystified-shouldqueue-dispatchable-interactswithqueue-queueable-serializesmodels/